### DISCUSSION TITLE
选集区域优化 部分功能无效/功能效果不明

### DISCUSSION THREAD

**Discussion by pinaZC** [the1812/Bilibili-Evolved#3766] [提问 / Q&A]
脚本版本：v2.5.1-15-g3869a146c
Commit: 3869a146c 内部名称: fullEpisodeTitle
浏览器版本：Microsoft Edge 版本 106.0.1370.52 (正式版本) (64 位)
播放器版本：3.20.11

1.展开选集标题
功能开启前后无明显效果变化（个人理解效果应为`展开动态标题`功能的效果，实际效果见截图）
![image](https://user-images.githubusercontent.com/72682556/197329863-a6c35a20-d789-486d-b1c8-8b6ef0f17606.png)

2.打开 `展开选集列表` 时, 在选集区域的标题上按住 Alt 键点击可以临时切换展开/收起选集列表.
实际操作效果为会下载一个名为 collectiondetail.htm 的文件，是否为功能失效？或者浏览器设置问题？或者个人操作问题？
![image](https://user-images.githubusercontent.com/72682556/197329991-bffba9ca-5b8a-434f-9da6-caa96942914b.png)
建议像弹幕列表一样加一个展开/收起按钮，或者在在功能面板中提供一个开关

**Comment by the1812** [comment]
视频发一下, 合集列表可能有好几种

**Reply by pinaZC** [reply]
https://www.bilibili.com/video/BV1M8411e7kA

**Reply by the1812** [reply]
![image](https://user-images.githubusercontent.com/26504152/197337040-35b1f61e-44a5-46a9-ab03-e99f05e469ca.png)
我这是正常的, 卸掉这个组件重新安装试试

**Reply by Geeyun-JY3** [reply]
我这边正式版预览版反复试了好几次，`展开选集标题` 确实对合集列表无效。

**Reply by Geeyun-JY3** [reply]
还有在选集区域的标题上按住 <kbd>Alt</kbd> 键点击可以临时切换展开/收起选集列表也是没试出来，多 P 和合集视频都是。

**Reply by the1812** [reply]
这就怪了

**Reply by Geeyun-JY3** [reply]
> ![image](https://user-images.githubusercontent.com/26504152/197337040-35b1f61e-44a5-46a9-ab03-e99f05e469ca.png)

这个是新版播放页还是旧版播放页，我这边测试结果是 `展开选集标题` 在旧版播放页无效，在新版播放页倒是有效。

**Reply by the1812** [reply]
是新版的

**Reply by Geeyun-JY3** [reply]
> 是新版的

那就对了。

那还适配旧版播放页吗（

**Reply by the1812** [reply]
旧版现在只适配合集类（稍后再看那种）
但那里并没有这种单独的选集区

**Reply by Geeyun-JY3** [reply]
> 旧版现在只适配合集类（稍后再看那种）
> 但那里并没有这种单独的选集区

额啥意思？
稍后再看那种合集类（URL 为 `https://www.bilibili.com/medialist/play/*`？）不是该组件的主要使用场景吧。
「合集类」不应该是指 https://github.com/the1812/Bilibili-Evolved/discussions/3766#discussioncomment-3939088 这种吗🤦‍♂️？（下文提到的「合集类」均指这种）

而且旧版播放页的分 P 类两个功能都能用啊。
目前是 `展开选集标题` 无法在旧版播放页的合集类视频中生效，`展开选集列表` 能在旧版播放页的合集类视频中生效。

**Reply by the1812** [reply]
就是这次各种新功能都没带旧版播放页玩的意思

**Reply by Geeyun-JY3** [reply]
> 就是这次各种新功能都没带旧版播放页玩的意思

行吧，明白了
